### ISSUE TITLE
Timeline API x-axis y-coordinates property

### ISSUE THREAD

**Issue by leducquan** [cerner/carbon-graphs#238]
Request:
A property on the timeline API to allow consumers to change the y-coordinate of the x-axis relative to the svg container.

Reason:

```
.carbon-axis {
  transform: translate(0px, -3px)
}
```
Above css styling works in Chrome, but does not work in IE as it does not support transformations of svg elements.

**Comment by Veera81** [comment]
Could you provide the screenshot? and code for reproduce the issue?

**Comment by leducquan** [comment]
I am using Windows.

For simplicity's sake, I've used https://engineering.cerner.com/carbon-graphs/#/timeline/simple as an example.
 
Original Timeline implementation
![image](https://user-images.githubusercontent.com/19377430/84717729-9e943d80-af6e-11ea-9d0c-eb4e1159e1c5.png)

An example of what we are trying to achieve using CSS transformation in CHROME (works)
![image](https://user-images.githubusercontent.com/19377430/84717825-e915ba00-af6e-11ea-8f11-8b62b0a725ed.png)

Not working in IE
![image](https://user-images.githubusercontent.com/19377430/84717937-46aa0680-af6f-11ea-8fc8-d89f089240d6.png)

It is a known issue that CSS transformation on svg elements are not supported for IE.  Firebox seems to be supported but have not tested. https://caniuse.com/#search=transform

So as a consumer of carbon timeline api, is there currently any property that allows me to move the x-axis up or down as per the screenshot?

**Comment by Veera81** [comment]
@leducquan What's the reason to move x-axis up or down? Is there any specific requirement?

**Comment by leducquan** [comment]
@Veera81 Yes, there is a requirement for our Event timeline to move the x-asis up slightly from the original position as we require custom positioning. Please see https://jira2.cerner.com/browse/UKIPDEV-1234 for more details.

**Comment by Veera81** [comment]
@leducquan Thanks for providing the details.

Have you tried other than transform functions?

**Comment by leducquan** [comment]
@Veera81 , what other way do you suggest?

**Comment by Veera81** [comment]
@leducquan Can we close this issue? Looks like the JIRA is closed

**Comment by leducquan** [comment]
no please keep this open, we still need a property exposed to us to allow us to shift the x-axis up or down.

**Comment by erinplourde** [comment]
This is needed to help allow the event timeline to show lines connected to the plotted symbols. The event timeline needs to be moved up so that there is enough room to place an event lines beneath. 

Please see callouts below: 

![image](https://user-images.githubusercontent.com/52933964/92135374-4e3fb580-edd0-11ea-9afe-66fbe20a4be8.png)

**Comment by mraplumb** [comment]
@leducquan the design provided above by @erinplourde is the same as requested for issue #225.  Would this meet your needs if we provide the functionality to opt into the shift/padding and the lines separately?  In your original screenshots, the shapes were rendered below the eventline.  Is this required for your use case?  We would prefer keeping the shapes on the event line as noted in the design above.

**Comment by leducquan** [comment]
@mraplumb Would be great. Please keep the shapes on the event line as per the design posted by @erinplourde , my above example was just to illustrate that we needed a property to allow us to shift the event line up or down.

**Comment by AshishMotanamGurunadham** [comment]
@erinplourde Looks like his exact requirement in this issue is to add an extra property which will move axis up and down as shown in below image. 

**Simple Timeline graph:** 
![image](https://user-images.githubusercontent.com/63670637/95756856-f7998700-0c6b-11eb-9112-fa9510f3582b.png)

**What consumer is expecting from us:**

![image](https://user-images.githubusercontent.com/63670637/95756657-ae493780-0c6b-11eb-9569-9a85bf1f1b7d.png)

Please let us know UX team thoughts on this issue.

**Comment by mraplumb** [comment]
@erinplourde to clarify, the request from @leducquan appears to keep the shapes **below** the timeline which is different from the visual you previously provided where the shapes remain on the timeline.

**Comment by erinplourde** [comment]
@mraplumb @AshishMotanamGurunadham  Hmm I see... Is there a way we are able to move the axis with the symbols? I would like to prevent consumers from plotting the symbols below the timeline but I understand what the stella team is trying to achieve.

**Comment by mraplumb** [comment]
I'll let @AshishMotanamGurunadham correct me if I'm wrong but I believe the consumer would provide use the same information they do today but could provide an additional value which moves the timeline up or down.  Technically speaking, we can move the timeline independent of the shapes or move them together. 

If we keep the shapes on the timeline, are we really just talking about extending the distance between the timeline and the tic values (i.e. 10am, 11am...etc.)?

**Comment by AshishMotanamGurunadham** [comment]
> @mraplumb @AshishMotanamGurunadham Hmm I see... Is there a way we are able to move the axis with the symbols? I would like to prevent consumers from plotting the symbols below the timeline but I understand what the stella team is trying to achieve.

@erinplourde Yes we can able to move the axis with symbols. But as you know looks like stella team only want to move axis up and down. 

Even if we implement a feature which move symbols with aixs, i think it won't be much useful for stella team. Correct me if am wrong.

**Comment by mraplumb** [comment]
After discussion, we have decided to make an update to allow increasing the padding below the timeline graph, but the shapes will remain on the timeline itself.
